app/src/renderer/components/common/TmSessionSignIn.vue

```diff
@@ -117,22 +104,22 @@ export default {
     async onSubmit() {
       this.$v.$touch()
       if (this.$v.$error) return
-      try {
-        await this.$store.dispatch(`testLogin`, {
-          password: this.fields.signInPassword,
-          account: this.fields.signInName
-        })
+      let passwordCorrect = await this.$store.dispatch(`testLogin`, {
```

Comment:
this should be `sessionCorrect`. Its a good practice to not give hints if the username is correct but the password isn't